gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -4,53 +4,91 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
+import React, { useState } from 'react';
 
 import { NxCheckbox } from '@sonatype/react-shared-components';
 import { NxRadio } from '@sonatype/react-shared-components';
-
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
-import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { NxButton } from '@sonatype/react-shared-components';
+import { NxStatefulTextInput } from '@sonatype/react-shared-components';
+import { NxErrorAlert } from '@sonatype/react-shared-components';
 
 export default function NxFormLayoutExample() {
+  function validator(val: string) {
+    return val.length ? null : 'Must be non-empty';
+  }
+
+  const [isRed, setIsRed] = useState(false),
+      [isBlue, setIsBlue] = useState(false),
+      [isGreen, setIsGreen] = useState(false),
+      toggleRed = () => setIsRed(!isRed),
+      toggleBlue = () => setIsBlue(!isBlue),
+      toggleGreen = () => setIsGreen(!isGreen);
+
+  const [color, setColor] = useState<string | null>(null);
+
+  // function onSubmit(evt) {
+  //   evt.preventDefaul();
+  //   alert('Submitted!');
+  // }
+
   return (
-    <form className="nx-form">
-      <div className="nx-form-row">
-        <div className="nx-form-group">
-          <label className="nx-label">
-            <span className="nx-label__text">Horizontal layout</span>
-            <span className="nx-sub-label">
-              <NxFontAwesomeIcon icon={faCalendar}/>
-              <span>Sub-label with icon.</span>
-            </span>
-            <input type="text" className="nx-text-input"/>
-          </label>
+    <form className="nx-tile">
+      <div className="nx-tile-content">
```

Comment:
should this be `form.nx-form` with no `div.nx-tile-content`?

Reply:
> should this be `form.nx-form` with no `div.nx-tile-content`?

The idea here was to create an example of a tile that consists of nothing but a form. In this instance we want the tile's look and feel hence the use of the tile header, content and footer. This example highlighted some discrepancies in the footers which need to be resolved.

Reply:
> > should this be `form.nx-form` with no `div.nx-tile-content`?
> 
> The idea here was to create an example of a tile that consists of nothing but a form. In this instance we want the tile's look and feel hence the use of the tile header, content and footer. This example highlighted some discrepancies in the footers which need to be resolved.

I removed the example here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/af777406b35c1158e36713e2935d2b2a6b76a5da